Makefile

```diff
@@ -59,21 +59,24 @@ check:
 	$(MAKE) test
 
 test:
-	$(CARGO) $(CARGO_OPTS) test
+	RUST_TEST_TASKS=1 RUST_BACKTRACE=1 GIT_COMMITTER_NAME="Chef CI" GIT_AUTHER_NAME="Chef CI"	EMAIL="blackhole@chef.io"	$(CARGO) $(CARGO_OPTS) test
```

Comment:
I think we could add some of these environment variables and include them in `CARGO_ENV`. I don't want to delay this so I can take care of it on my next PR. Just wanted to mention it.